plumber-android-core/src/main/java/leakcanary/ViewLocationHolderLeakFix.kt

```diff
@@ -26,6 +30,16 @@ object ViewLocationHolderLeakFix {
     if (VERSION.SDK_INT != 28) {
       return
     }
+    // Takes care of child windows (e.g. dialogs)
+    Curtains.onRootViewsChangedListeners += OnRootViewRemovedListener {
+      if (Looper.getMainLooper().thread === Thread.currentThread()) {
```

Comment:
Link #2262, we can add `isMainThread` to reuse thread checking?

Reply:
yep

Reply:
https://github.com/square/leakcanary/pull/2263